user/quick-start.rst

```diff
@@ -124,8 +124,10 @@ Before building the workspace with ``colcon`` the steps also fetch some addition
     $ colcon metadata update
     $ colcon build
 
-To run Gazebo which requires environment variables for e.g. the model paths the same commands as for other packages can be used.
-Using the additional metadata the source script will also automatically source the Gazebo specific file ``share/gazebo/setup.sh`` which defines these environment variables.
+This last step also allows Gazebo to read environment variables during exexucution (e.g.,
+``GAZEBO_MODEL_PATH``). This is possible by sourcing the ``install/local_setup.bash`` script
+(adjust accordingly for different shells)  which in turn delegates to the Gazebo specific
+``setup.sh`` file.
```

Comment:
I am not sure the updated paragraph is clearer. E.g. the phrase "allows Gazebo to read environment variables during exexucution" sounds weird. By implicitly sourcing the setup file from Gazebo it is made sure that the Gazebo-specific environment variables will be set. Nothing changes how Gazebo tries to read / use them. Also pointing to the full path of the Gazebo-specific setup file is valuable and shouldn't be removed imo.

Reply:
Yeah, taking a second look at it, my updates don't make it any clearer; I'll revert it